cmd/jaeger/integration/goldendataset/pict_input_traces.txt

```diff
@@ -0,0 +1,3 @@
+Resource: Empty, VMOnPrem, VMCloud, K8sOnPrem, K8sCloud, Faas, Exec
```

Comment:
what is the meaning of the "pict_" prefix?

Reply:
PICT stands for Pairwise Independent Combinatorial Tool (https://github.com/microsoft/pict/), apparently correctnesstest in testbed uses this tool as its source of combinatoric data. Here's the example e2e test with correctnesstest: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/testbed/correctnesstests/traces/correctness_test.go.

Reply:
Testbed has 3 data provider types to generate the testdata:
1. perfTestDataProvider: randomly generate traces that might not cover all trace combinations.
2. goldenDataProvider: generate traces from PICT generated file with the purpose of covering all combinatorial trace data.
3. fileDataProvider: parse provided traces from a file in yaml or json format.

I decided to use `goldenDataProvider` because `perfTest` and `file` data providers are created for use in performance tests.

Reply:
> also, why "golden"? Usually we place test input files under ./fixtures/

Because the testbed example stores the test input files under the 'goldendataset/' directory, I'll change that to align with our convention.